From: David Sterba <dsterba@suse.cz>
Date: Thu, 24 Apr 2014 15:09:10 +0200
Patch-mainline: 3.16
Git-commit: 7d6213c5a764d0591afeb9faa3f7ee18e8e74c25
Subject: [PATCH] btrfs: make DEV_INFO ioctl available to anyone

This ioctl provides basic info about the devices that can be obtained in
other ways (eg. sysfs), there's no reason to restrict it to
CAP_SYSADMIN.

Signed-off-by: David Sterba <dsterba@suse.cz>
Signed-off-by: Chris Mason <clm@fb.com>
---
 fs/btrfs/ioctl.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/fs/btrfs/ioctl.c
+++ b/fs/btrfs/ioctl.c
@@ -2459,9 +2459,6 @@ static long btrfs_ioctl_dev_info(struct
 	char *s_uuid = NULL;
 	char empty_uuid[BTRFS_UUID_SIZE] = {0};
 
-	if (!capable(CAP_SYS_ADMIN))
-		return -EPERM;
-
 	di_args = memdup_user(arg, sizeof(*di_args));
 	if (IS_ERR(di_args))
 		return PTR_ERR(di_args);
